### PR TITLE
TASK: Refactor the ``FusionView`` to use fusion in variable and method names

### PR DESCRIPTION
**Pull request by mficzel**
The previous method are move into the ``TypoScriptView`` that maps them
to the new ones. Also the ``setOption`` and ``getOption`` of the ``TypoScriptView`` still
support the `typoScript*`-options but map them internally.

### CODE REVIEW

File: Neos.Fusion/Classes/View/FusionView.php [hunk 1]
```diff
@@ -268,16 +268,6 @@ protected function getFusionPathForCurrentRequest()
     }
 
     /**
-     * Determine whether we are able to find Fusion at the requested position
-     *
-     * @return boolean TRUE if Fusion exists at the current Fusion path; FALSE otherwise
-     */
-    protected function isTypoScriptFoundForCurrentRequest()
```
Comment (mficzel) [inline Neos.Fusion/Classes/View/FusionView.php]: I could not find any use of this method so i simply removed it.

File: Neos.Fusion/Classes/View/TypoScriptView.php [hunk 2]
```diff
@@ -18,4 +18,88 @@
...
 class TypoScriptView extends FusionView
 {
+    /**
+     * Sets the TypoScript path to be rendered to an explicit value;
+     * to be used mostly inside tests.
+     *
+     * @param string $typoScriptPath
+     * @return void
+     */
+    public function setTypoScriptPathPath($typoScriptPath)
```
Comment (robertlemke) [inline Neos.Fusion/Classes/View/TypoScriptView.php]: Caution: setTypoScript~~Path~~Path()

File: Neos.Fusion/Classes/View/TypoScriptView.php [hunk 3]
```diff
@@ -18,4 +18,88 @@
...
+    {
+        parent::setFusionPath($typoScriptPath);
+    }
+
+    /**
+     * @param string $pathPattern
+     * @return void
+     */
+    public function setTypoScriptPathPathPattern($pathPattern)
```
Comment (robertlemke) [inline Neos.Fusion/Classes/View/TypoScriptView.php]: Caution: setTypoScript~~Path~~PathPattern()

File: Neos.Fusion/Classes/View/TypoScriptView.php [hunk 4]
```diff
@@ -18,4 +18,88 @@
...
+    {
+        parent::setFusionPathPattern($pathPattern);
+    }
+
+    /**
+     * @param array $pathPatterns
+     * @return void
+     */
+    public function setTypoScriptPathPathPatterns(array $pathPatterns)
+    {
```
Comment (robertlemke) [inline Neos.Fusion/Classes/View/TypoScriptView.php]: Caution: setTypoScript~~Path~~PathPatterns()

**Comment by robertlemke** [comment]
I'll fix these problems right away.

**Pull request merged**